pom.xml

```diff
@@ -134,6 +134,12 @@
         <version>4.11.0</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-junit-jupiter</artifactId>
+        <version>2.17.0</version>
```

Comment:
IIUC, this is part of the migration to JUnit 5 where the `@ExtendWith(MockitoExtension.class)` annotations are split out to an extension package.

I think this is fine as I also noticed that we don't manage mockito v5 upstream so I think it should be fine to override it here for now. qq, I don't think this is the latest version: https://central.sonatype.com/artifact/org.mockito/mockito-junit-jupiter/5.21.0

Reply:
Yeah, you are rightly deduced - need this dependency for `MockitoExtension.class`.
Double checked and sadly also couldn't find a managed version anywhere